src/styles.css

```diff
@@ -34,7 +34,7 @@ button {
 }
 
 button:hover {
-  background: #FCE49E;
```

Comment:
Maybe not as 'pretty' a yellow with our other colors, but we believe this is a higher contrast and is better for accessibility.